sdk/core/azure-core-http-netty/README.md

```diff
@@ -34,6 +34,8 @@ The following sections provide several code snippets covering some of the most c
 Create a Netty Http client that uses port 80 and has no proxy.
 
 ```java
+// ./src/samples/java/com/azure/core/http/netty/ReadmeSamples.java#L19-L19
```

Comment:
Is there a way to hide this in the README?

Reply:
There is a way to hide but I like to leave it as it is so that we'll know where the snippets are coming from and leaving it here will allow for future updates to automatically pick up the changes.